### ISSUE TITLE
IntroduceVariableCodeRefactoringProvider throws ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by Gnbrkm41** [dotnet/roslyn#31495]
**Version Used**: 
Visual Studio 2017 Community 15.9.3 with ReSharper Ultimate 2018.2.3
**Steps to Reproduce**:
Was able to reproduce with the following code:
```cs
using System;

namespace ConsoleApp11
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
        }
    }

    public class SomeType
    {
        private static string 

    }
}
```
Note that there is a whitespace after `string`. Try selecting from the point after the whitespace to next line.

**Expected Behavior**:
Unfortunately not sure what's supposed to happen, but definitely shouldn't throw exceptions xD

**Actual Behavior**:
Throws an exception.

![ezgif-1-55b29bbfd1ff](https://user-images.githubusercontent.com/42944058/49342580-992f1500-f6a0-11e8-9928-93891858fb7c.gif)

Stack trace:
```
System.ArgumentOutOfRangeException : 'end' must not be less than 'start'
Parameter name: end
   at Microsoft.CodeAnalysis.Text.TextSpan.FromBounds(Int32 start,Int32 end)
   at Microsoft.CodeAnalysis.IntroduceVariable.AbstractIntroduceVariableService`6.State.GetExpressionUnderSpan(SyntaxTree tree,TextSpan textSpan,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.IntroduceVariable.AbstractIntroduceVariableService`6.State.TryInitialize(TextSpan textSpan,CancellationToken cancellationToken)
   at Microsoft.CodeAnalysis.IntroduceVariable.AbstractIntroduceVariableService`6.State.Generate(TService service,SemanticDocument document,TextSpan textSpan,CancellationToken cancellationToken)
   at async Microsoft.CodeAnalysis.IntroduceVariable.AbstractIntroduceVariableService`6.IntroduceVariableAsync[TService,TExpressionSyntax,TTypeSyntax,TTypeDeclarationSyntax,TQueryExpressionSyntax,TNameSyntax](<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeRefactorings.IntroduceVariable.IntroduceVariableCodeRefactoringProvider.ComputeRefactoringsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeRefactorings.CodeRefactoringService.GetRefactoringFromProviderAsync(<Unknown Parameters>)
```

**Comment by sharwell** [comment]
Duplicate of #27949

**Comment by Gnbrkm41** [comment]
Sorry to bother, thought it was a different one as other similar issues were closed but it was keep happening, but apparently seems the fix will be shipped in the next version. Cheers!

**Comment by jinujoseph** [comment]
No worries @Gnbrkm41 , the fix for this should be available in 16.0.Preview1 (shipped today ) onwards.
